tests/test_python.py

```diff
@@ -96,9 +97,17 @@ def test_python_webserver_1(container_per_test):
     ).is_listening
 
     # start of the python http server
-    bci_pyt_serv = container_per_test.connection.run_expect(
+    container_per_test.connection.run_expect(
         [0], f"timeout 240s python3 -m http.server {port} &"
-    ).stdout
+    )
+
+    # wait for the webserver to start
+    timeout = 60
+    for i in range(timeout + 1):
+        assert i < timeout, "webserver startup timeout"
+        if container_per_test.connection.run(f"nc -z localhost {port}").rc:
+            break
+        time.sleep(1)
```

Comment:
and `i++`?

Reply:
`for i in range(timeout + 1):` in line 106